docs/source/contribution.rst

```diff
@@ -431,15 +431,21 @@ To build the documentation, you need to install `Sphinx <http://www.sphinx-doc.o
 
   $ pip install sphinx sphinx_rtd_theme
 
-Then you can build the documentation in HTML format locally::
+Docstrings (documentation comments in the source code) are collected from the installed Chainer module.
+In order to generate the html files, Chainer must be installed in develop mode. If Chainer has not been installed in develop mode, then ``make html`` will yield an error that the reference sources cannot
```

Comment:
>Chainer must be installed in develop mode

This statement is not correct either: if Chainer is installed in non-develop (more precisely, non-editable) mode, modules are collected from "installed" files. The build process works just fine. The problem is that if a developer modifies files in the checked-out source files, which are not "installed" ones, the edits will not be reflected.
Let's be more precise and clear.

Reply:
Edited accordingly.

Reply:
Hmm, is this comment reflected?
It's still saying "Chainer must be installed in develop mode".

Reply:
Yes. "In order to see changes in checked-out source files..." In other words, only if you want to see local changes, do you need to install in develop mode.

Reply:
Thanks, I confirmed.
I don't know why I gave the above comment... perhaps I was looking at a wrong diff.